app/src/main/java/com/appandweb/collapsingtoolbar/ui/activity/ViewPagerActivity.java

```diff
@@ -0,0 +1,192 @@
+import com.appandweb.collapsingtoolbar.ui.presenter.ViewPagerPresenter;
+import com.appandweb.collapsingtoolbar.ui.presenter.abs.AbsViewPagerPresenter;
+import com.appandweb.collapsingtoolbar.ui.adaptor.ViewPagerAdapter;
+import com.appandweb.collapsingtoolbar.ui.fragments.SecondFragment;
+import com.appandweb.collapsingtoolbar.ui.fragments.ThirdFragment;
+import com.appandweb.collapsingtoolbar.ui.fragments.FirstFragment;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import butterknife.Bind;
+import butterknife.BindColor;
+import butterknife.BindDrawable;
+import butterknife.BindString;
+import butterknife.OnClick;
+
+public class ViewPagerActivity extends BaseActivity implements TabLayout.OnTabSelectedListener, AbsViewPagerPresenter.MVPView, AbsViewPagerPresenter.Navigator, PopupMenu.OnMenuItemClickListener {
+    @Bind(R.id.view_pager_ibtn_back)
+    ImageButton btn_back;
+    @Bind(R.id.view_pager_ibtn_menu)
+    ImageButton btn_menu;
+    @Bind(R.id.view_pager_tablayout)
+    TabLayout tabLayout;
+    @Bind(R.id.view_pager_vp)
+    ViewPager viewPager;
+    @Bind(R.id.view_pager_tv_title)
+    TextView title_header;
+
+    @BindDrawable(R.drawable.home_dark)
+    Drawable dwb_home;
+    @BindDrawable(R.drawable.message_light)
+    Drawable dwb_message;
+    @BindDrawable(R.drawable.inbox_light)
+    Drawable dwb_inbox;
+    @BindColor(R.color.colorPrimaryDark)
+    int ppColor;
+    @BindColor(android.R.color.darker_gray)
+    int ssColor;
+    @BindString(R.string.first_fragment)
+    String first_fragment;
+    @BindString(R.string.second_fragment)
+    String second_fragment;
+    @BindString(R.string.third_fragment)
+    String third_fragment;
+
+    @OnClick(R.id.view_pager_ibtn_menu)
+    public void onMenuClicked(View view) {
+        presenter.onMenuClicked();
+    }
+
+    @OnClick(R.id.view_pager_ibtn_back)
+    public void onBackPressed(View view) {
+
+        presenter.onBackClicked();
+    }
+
+    AbsViewPagerPresenter presenter;
+    ViewPagerAdapter adapter;
+    PopupMenu menu;
+    private static final int FIRST = 0;
+    private static final int SECOND = 1;
+    private static final int THIRD = 2;
+
+    @Override
+    protected void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        initialize();
+    }
+
+    private void initialize() {
+        presenter = new ViewPagerPresenter(this);
+        presenter.setNavigator(this);
+        presenter.setView(this);
+
+        adapter = new ViewPagerAdapter(getSupportFragmentManager(), getViewPagerFragments());
+        viewPager.setAdapter(adapter);
+        tabLayout.setupWithViewPager(viewPager);
+        tabLayout.setOnTabSelectedListener(this);
+        tabLayout.getTabAt(0).setIcon(dwb_message);
+        tabLayout.getTabAt(1).setIcon(dwb_home);
+        tabLayout.getTabAt(2).setIcon(dwb_inbox);
+        viewPager.setCurrentItem(1);
+
+        menu = new PopupMenu(this, btn_menu);
```

Comment:
would be nice to extract these lines to a method, something like `configurePopupMenu()`